### PR TITLE
React Apollo Hooks tutorial

### PR DESCRIPTION
**Pull request by vardhanapoorv**
Tutorial with app-boilerplate, app-final and tutorial-site

### CODE REVIEW

File: tutorials/frontend/react-apollo-hooks/tutorial-site/config.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+			}
+		],
+		"frontline": false,
+		"ignoreIndex": true
+	},
+	"siteMetadata": {
+		"title": "2 hour GraphQL course for React developers | Hasura",
+		"description": "A concise and powerful tutorial that covers fundamental concepts of both GraphQL and using GraphQL in React",
+		"ogImage": "https://graphql-engine-cdn.hasura.io/learn-hasura/assets/social-media/twitter-card-react-apollo-hooks.png",
+		"docsLocation": "https://github.com/hasura/graphql-engine/tree/master/community/learn/graphql-tutorials/tutorials/react-apollo-hooks/tutorial-site/content",
```
Comment (praveenweb) [inline tutorials/frontend/react-apollo-hooks/tutorial-site/config.js]: Update docs location to point to this repo

File: tutorials/frontend/react-apollo-hooks/tutorial-site/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+#### Build Docker Image
+```bash
+docker build -t tutorial-site:0.1 -f Dockerfile.localdev .
+```
+
+#### Run Container in Dev Mode
+
+```bash
+docker run -ti -p 8080:8080 -v /path/to/graphql-engine/community/learn/graphql-tutorials/tutorials/react-apollo-hooks/tutorial-site/content:/gatsby-gitbook-starter/content -v /path/to/graphql-engine/community/learn/graphql-tutorials/tutorials/react-apollo-hooks/tutorial-site/config.js:/gatsby-gitbook-starter/config.js tutorial-site:0.1
```
Comment (praveenweb) [inline tutorials/frontend/react-apollo-hooks/tutorial-site/README.md]: update new repo path

Comment (vardhanapoorv) [reply tutorials/frontend/react-apollo-hooks/tutorial-site/README.md]: Done

**Comment by marionschleifer** [comment]
@vardhanapoorv could you please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

**Pull request merged**